CHANGELOG.md

```diff
@@ -28,8 +29,7 @@
 
 ### Breaking changes
 
-For the existing backend mode users who have table visualization might see some inconsistencies on their panels. We have updated the table column naming. This will potentially affect field transformations and/or field overrides. To resolve this either:
-
+For the existing backend mode users who have table visualization might see some inconsistencies on their panels. We have updated the table column naming. This will potentially affect field transformations and/or field overrides.  To resolve this either:
```

Comment:
```suggestion
For the existing backend mode users who have table visualization might see some inconsistencies on their panels. We have updated the table column naming. This will potentially affect field transformations and/or field overrides. To resolve this either:
```

Reply:
This is unfortunately a bug in the current update-changelog action